server/scsynth/scsynth_main.cpp

```diff
@@ -80,6 +80,7 @@ void Usage() {
              "          UDP ports never require passwords, so for security use TCP.\n"
              "   -N <cmd-filename> <input-filename> <output-filename> <sample-rate> <header-format> <sample-format>\n"
 #ifdef __APPLE__
+             "   -s <safety-clip-threshold>          (default %d)\n"
```

Comment:
you should definitely add a note here that 'inf' (or negative number if you agree with my comment above) will disable it. and maybe add a brief explanation of what the option does here too.